test/specs/components/Menu/MenuItem-test.tsx

```diff
@@ -29,4 +28,136 @@ describe('MenuItem', () => {
     expect(menuItem.find('.ui-menu__item').is('li')).toBe(true)
     expect(menuItem.text()).toBe('Home')
   })
+
+  describe('accessibility', () => {
+    handlesAccessibility(MenuItem, { defaultRootRole: 'presentation' })
+    handlesAccessibility(MenuItem, { defaultRootRole: 'menuitem', partSelector: 'a' })
+
+    describe('as a ToolbarButton', () => {
+      test('root role should be presentation', () => {
+        const menuItemComponent = getTestingRenderedComponent(
+          MenuItem,
+          <MenuItem accessibility={ToolbarButtonBehavior} />,
+        )
+        expect(getRenderedAttribute(menuItemComponent, 'role', '')).toBe('presentation')
+      })
+
+      test('anchor role should be button', () => {
+        const menuItemComponent = getTestingRenderedComponent(
+          MenuItem,
+          <MenuItem accessibility={ToolbarButtonBehavior} />,
+        )
+        expect(getRenderedAttribute(menuItemComponent, 'role', 'a')).toBe('button')
+      })
+
+      test('aria-label should be added to the child anchor', () => {
+        const ariaLabel = 'Useful Tool Tip'
+        const menuItemComponent = getTestingRenderedComponent(
+          MenuItem,
+          <MenuItem aria-label={ariaLabel} accessibility={ToolbarButtonBehavior} />,
+        )
+
+        expect(getRenderedAttribute(menuItemComponent, 'aria-label', '')).toBe(undefined)
+        expect(getRenderedAttribute(menuItemComponent, 'aria-label', 'a')).toBe(ariaLabel)
+      })
+
+      test('aria-labelledby should be added to the child anchor', () => {
+        const ariaLabelledByID = 'element-that-labels'
+        const menuItemComponent = getTestingRenderedComponent(
+          MenuItem,
+          <MenuItem aria-labelledby={ariaLabelledByID} accessibility={ToolbarButtonBehavior} />,
+        )
+
+        expect(getRenderedAttribute(menuItemComponent, 'aria-labelledby', '')).toBe(undefined)
+        expect(getRenderedAttribute(menuItemComponent, 'aria-labelledby', 'a')).toBe(
+          ariaLabelledByID,
+        )
+      })
+
+      const disabledFlags = [true, false]
+      disabledFlags.forEach(disabledValue => {
+        test(`aria-disabled should be ${disabledValue} if button disabled prop is ${disabledValue}`, () => {
```

Comment:
seems that there is a error here, as we are setting `disabled` for `Menu`, not for `Button`